src/foam/blob/Blob.js

```diff
@@ -22,13 +22,11 @@ foam.CLASS({
 
   properties: [
     {
-      class: 'Long',
+      class: 'Int',
```

Comment:
Switching these all to Ints will limit us to Blobs under 4GB will it not?  Why the change?

Reply:
I was noticing in a lot of cases previously that I was casting down to an integer anyway because the read & write functions of InputStream / OutputStream respectively don't take long's as the offset and length parameters. But beyond that do we ever envision having to upload a file larger than 4GB? If that's the case then we can modify this.

Reply:
I have use cases in other work and at home where I'll want to deal with larger files :)